apps/server-asset-sg/src/features/users/user.model.ts

```diff
@@ -1,37 +1,23 @@
-import { Role as PrismaRole } from '@prisma/client';
-import { IsArray, IsBoolean, IsEnum, IsString } from 'class-validator';
+import { IsArray, IsBoolean, IsString } from 'class-validator';
+import { Role } from '@/features/workgroups/workgroup.model';
 import { Data, Model } from '@/utils/data/model';
 
 export interface User extends Model<UserId> {
   email: string;
-  role: Role;
   lang: string;
   workgroups: WorkgroupOnUser[];
   isAdmin: boolean;
 }
 
 export interface WorkgroupOnUser {
-  workgroupId: number;
-  role: PrismaRole;
-  workgroup: {
```

Comment:
shouldnt we keep the name for easier handling in the frontend?

Reply:
I'm currently expecting that we will load all available workgroups via reference-data, which would mean that we don't need the name in here, as we can get the whole workgroup via its id.